wafer/schedule/templates/wafer.schedule/edit_schedule.html

```diff
@@ -121,8 +129,12 @@
             <span draggable="true" class="col-md-6 badge badge-warning draggable"
                   id="talk{{ talk.talk_id }}"
                   data-toggle="tooltip" data-placement="left"
-                  title="{{ talk.title }}" data-type="talk" data-talk-id="{{ talk.talk_id }}">
-              {{ talk.title|truncatechars:24 }}
+                  title="{{ talk.title }} (Cancelled)" data-type="talk" data-talk-id="{{ talk.talk_id }}">
+              {% if not talk.cancelled %}
+              {{ talk.title|truncatechars:24 }} Isn't cancelled
```

Comment:
I'm not following why we need the "Isn't cancelled" here?

Reply:
That was left over debugging output - removed.